### DISCUSSION TITLE
Listeners By Day Snapshot in Statistics Overview

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4054] [Q&A]
**Using Docker installation method**

Yes

**AzuraCast version**

Rolling Release #aea1cde

**Host Operating System**

Ubuntu 20.04

**Describe the bug**

Is there any way to change the time that the Listeners By Day snapshot is taken in the Statistics Overview? My snapshot is taken at 4:00 am EST, which is hardly a peak listening time in the U.S. and not an accurate reflection of my listeners. 

**To Reproduce**

**Expected behavior**

**Relevant Logs**

**Screenshots**

**Device(s):**

- Device: 
- OS: 
- Browser: 
- Version: 

**Additional context**

**Comment by dekakast** [comment]
I have the same issue. My "Listener by Day" snapshot in Statistics Overview is taken at 6 am.
